### PR TITLE
feat: provide utility functions for polygon migration

### PR DESCRIPTION
**Pull request by goga-m**
## Summary
A set of utility functions that provide required variables for the migration

* `#migrationNetwork` - Uses devnet when in preview mode
* `#migrationTransactionFee`
* `#metamaskUrl`
* `#migrationWalletAddress`

Part of https://app.clickup.com/t/86773x6yg

Migration flow can be tested at https://github.com/ArdentHQ/arkvault/pull/292

## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [x] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: src/utils/polygon-migration.test.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import {
+	migrationTransactionFee,
+	migrationGuideUrl,
+	metamaskUrl,
+	migrationNetwork,
+	migrationWalletAddress,
+} from "./polygon-migration";
+
+describe("Url validation", () => {
```
Comment (ItsANameToo) [inline src/utils/polygon-migration.test.ts:9]: url validation? 🤔

File: .env.example [hunk 2]
```diff
@@ -1 +1,5 @@
 VITE_POLYGON_MIGRATION_TRANSACTION_FEE=0.05
+VITE_MIGRATION_GUIDE_URL=https://arkvault.io/docs
+VITE_MIGRATION_ADDRESS=DNBURNBURNBURNBRNBURNBURNBURKz8StY
+VITE_MIGRATION_NETWORK=ark.mainnet
```
Comment (ItsANameToo) [inline .env.example:4]: can you change it to default to `ark.devnet` so it's in line with the default burn address as well

File: src/utils/polygon-migration.test.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	it("#migrationTransactionFee", () => {
+		expect(migrationTransactionFee()).toBe(0.05);
+	});
+
+	it("migrationGuideUrl", () => {
```
Comment (ItsANameToo) [inline src/utils/polygon-migration.test.ts:14]: keep the names consistent with `#` in front

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

**Comment by goga-m** [comment]
@ItsANameToo thanks for checking. The issues should be resolved

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**